custom-namespaces.yaml

```
@@ -571,6 +571,13 @@ firefox_accounts:
       type: table_view
       tables:
         - table: moz-fx-data-shared-prod.accounts_db.fxa_oauth_clients
+    monitor_db_counts:
+      type: table_view
+      tables:
+        - table: moz-fx-data-shared-prod.backend_derived.monitoring_db_counts_v1
+      measures:
+        count:
+          type: count
```

This will just count the rows in the query, but that isn't very useful information. I'm guessing you want to sum the `total_rows` column values instead.
```suggestion
        total_rows_sum:
          type: sum
          sql: ${total_rows}
```
(IMO it'd be nicer if the column in the underlying table/view didn't have "total" in the name, and instead was named something like `row_count`, then this measure could be something like `total_row_count`)